html/includes/modal/edit_alert_transport.inc.php

```diff
@@ -39,37 +39,18 @@
                             <div class="col-sm-3">
                                 <select name='transport-choice' id='transport-choice' class='form-control'>
-                                    <option value="api-form">API</option>
-                                    <option value="boxcar-form">Boxcar</option>
-                                    <option value="canopsis-form">Canopsis</option>
-                                    <option value="ciscospark-form">Cisco Spark</option>
-                                    <option value="clickatell-form">Clickatell</option>
-                                    <option value="discord-form">Discord</option>
-                                    <option value="elasticsearch-form">Elasticsearch</option>
-                                    <option value="gitlab-form">Gitlab</option>
-                                    <option value="hipchat-form">Hipchat</option>
-                                    <option value="irc-form">IRC</option>
-                                    <option value="jira-form">Jira</option>
-                                    <option value="kayako-form">Kayako Classic</option>
-                                    <option value="mail-form" selected>Mail</option>
-                                    <option value="msteams-form">Microsoft Teams</option>
-                                    <option value="nagios-form">Nagios</option>
-                                    <option value="opsgenie-form">OpsGenie</option>
-                                    <option value="osticket-form">osTicket</option>
-                                    <option value="pagerduty-form">PagerDuty</option>
-                                    <option value="hue-form">Phillips Hue</option>
-                                    <option value="playsms-form">PlaySMS</option>
-                                    <option value="pushbullet-form">Pushbullet</option>
-                                    <option value="pushover-form">Pushover</option>
-                                    <option value="rocket-form">Rocket.chat</option>
-                                    <option value="slack-form">Slack</option>
-                                    <option value="smseagle-form">SMSEagle</option>
-                                    <option value="smsfeedback-form">SMSFeedback</option>
-                                    <option value="syslog-form">Syslog</option>
-                                    <option value="telegram-form">Telegram</option>
-                                    <option value="twilio-form">Twilio</option>
-                                    <option value="victorops-form">Victorops</option>
-                                    <!--Insert more transport type options here has support is added. Value should be: [transport_name]-form -->
+<?php
+
+// Create list of transport
+$transport_dir = Config::get('install_dir').'/LibreNMS/Alert/Transport';
```

Comment:
We already do all of this later on. It would make more sense to store the results of `scandir()` and then re-use that later as well. Make sense?

Reply:
Done. Also removed duplicate code.